Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -28,6 +28,11 @@ public enum ModuleError: Swift.Error {
         case unexpectedSourceFiles([String])
         case modulemapInSources(String)
     }
+	
+	///	Describes a way in which a path is invalid.
```

Comment:
Please use only one space between `///` and `Describes`.

Reply:
fixed